webpack.config.js

```diff
@@ -98,7 +98,9 @@ const resolve = {
 spcPackageJson = require("@hms-dbmi-bgm/shared-portal-components/package.json");
 spcPeerDependencies = spcPackageJson.peerDependencies || {};
 Object.keys(spcPeerDependencies).forEach(function(packageName) {
-    resolve.alias[packageName] = path.resolve("./node_modules/" + packageName);
+    if (packageName !== 'auth0-lock') {
+        resolve.alias[packageName] = path.resolve("./node_modules/" + packageName);
+    }
```

Comment:
Looks good, am curious if now that we're on React 17 we can get rid of this Object.keys(...).foreEach(...) block entirely (not sure I tested it without this on React17), but not too important / very low priority.